db/dbformat.h

```diff
@@ -272,6 +278,30 @@ class LookupKey {
   Slice user_key() const
   { return Slice(kstart_, end_ - kstart_ - KeySuffixSize(internal_key())); }
 
+  // did requestor have metadata object?
+  bool WantsKeyMetaData() const {return(NULL!=meta_);};
+
+  void SetKeyMetaData(ValueType type, SequenceNumber seq, ExpiryTime expiry) const
+  {if (NULL!=meta_)
```

Comment:
Since this is a multi-line code block, should probably move the "if" to the next line. Same comment for the next method.